sql/telemetry_derived/clients_daily_keyed_boolean_aggregates_v1/query.sql

```diff
@@ -100,7 +104,7 @@ sampled_data AS (
     OR (channel = "release" AND os != "Windows")
     OR (channel = "release" AND os = "Windows" AND MOD(sample_id, @sample_size) = 0)
 ),
--- Using `min` for when `agg_type` is `count` returns null when all rows are null
+        -- Using `min` for when `agg_type` is `count` returns null when all rows are null
```

Comment:
I don't understand why this space is now getting injected where it didn't before. Could be dedent behavior changing in python versions?

Reply:
This is strange, I don't see any place where this behavior necessarily changed.

Reply:
Another explanation could be that the query was not updated when the code was, since the query is generated at runtime in the Airflow DAG.

Reply:
> Another explanation could be that the query was not updated when the code was, since the query is generated at runtime in the Airflow DAG

I think that's likely. I suspect this that the additional space is comes from that fact that the script calls `reformat` for our SQL formatting library. I believe that lib attempts to preserve indentation of comments while it radically changes indentation of surrounding code.

We're injecting various variables into the text here and I'd imagine they are not all indented consistently. Anyways, this is a small issue that I'm going to ignore.